enterprise/client-failover-cluster/src/main/java/sample/DeclarativeFailoverExample.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sample;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.LifecycleEvent;
+import com.hazelcast.core.LifecycleListener;
+
+public class DeclarativeFailoverExample {
+
+    public static void main(String[] args) {
+
+        HazelcastInstance client = HazelcastClient.newHazelcastFailoverClient(null);
+
+        //Client will by default logs as info
+        //INFO: hz.client_0 [cluster1] [3.12-SNAPSHOT] HazelcastClient 3.12-SNAPSHOT (20190205 - 3af10e3) is CLIENT_CHANGED_CLUSTER
```

Comment:
3.12-SNAPSHOT ???

Reply:
It is an example log. Whatever, I put here will be out-of-date. 
What do you think I should write? What do you mean by `3.12-SNAPSHOT ???` ?

Reply:
I see that you already removed the SNAPSHOT, which is good. I would prefer also the first sentence as: `Client will by default logs as info. The log will be similar to:`